p4-16/spec/P4-16-spec.mdk

```diff
@@ -3467,7 +3467,7 @@ Bit-strings also support the following operations:
   unchanged.  A slice of an unsigned integer is an unsigned integer.
 - Logical shift left and right with a runtime known unsigned integer
   value, denoted by `<<` and `>>` respectively. In a shift, the left operand is unsigned, and right operand must be either an
-  expression of type `bit<S>` or a non-negative integer literal.
+  expression of type `bit<S>` or a non-negative integer constant.
```

Comment:
@jnfoster @mbudiu-vmw Is this a case where you would prefer the phrase "compile time known value" instead of "constant"?  I do not have a strong preference, but wanted to see if you did.

Reply:
"compile time known value" is very verbose, but I think we consistently used it in the spec... it's perhaps more precise than "constant"

Reply:
I agree with this...